blockchain/Blast Bridge.ipynb

```diff
@@ -118,7 +118,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "AMBERDATA_API_KEY = 'AMBERDATA_API_KEY'  #Enter Amberdata API Key"
+    "AMBERDATA_API_KEY = 'UAK7698aee0ad864ec4d56cb3a42275a563'  #Enter Amberdata API Key"
```

Comment:
Mm, do we want to put an API key in Github? This is a public repo no?

Reply:
I always delete and save without a key.... why does it do this. Thanks for catching that. Ill fix